PWGLF/Tasks/cascadeanalysis.cxx

```diff
@@ -54,54 +54,64 @@ using std::array;
 
 struct cascadeQA {
   //Basic checks
-  OutputObj<TH1F> hMassXiMinus{TH1F("hMassXiMinus", "", 3000, 0.0, 3.0)};
-  OutputObj<TH1F> hMassXiPlus{TH1F("hMassXiPlus", "", 3000, 0.0, 3.0)};
-  OutputObj<TH1F> hMassOmegaMinus{TH1F("hMassOmegaMinus", "", 3000, 0.0, 3.0)};
-  OutputObj<TH1F> hMassOmegaPlus{TH1F("hMassOmegaPlus", "", 3000, 0.0, 3.0)};
+  HistogramRegistry registry{
```

Comment:
This is a great addition, maybe you could also define `axisSpec` variables to handle spectra with similar axes. Also, it would be good to explicitly state the units of the plot axis